Scripts/script-DemistoLinkIncidents.yml

```diff
@@ -0,0 +1,44 @@
+commonfields:
+  id: DemistoLinkIncidents
+  version: -1
+name: DemistoLinkIncidents
```

Comment:
Slavik just added option to do it from a script.ight be irrelevant.
If still relevant maybe worth adding option to duplicate.

Reply:
I know. but this might be out before the build, and it's a good example...